.eslintrc.json

```diff
@@ -187,6 +187,7 @@
       "error"
     ],
     "no-empty": "off",
-    "eol-last": "warn"
+    "eol-last": "warn",
+    "no-multiple-empty-lines": ["warn", { "max": 2 }]
```

Comment:
should `max` maybe be set to `1` instead of `2`?

i initially had tried this, but there were so many cases found where `2` were used and so i thought it is part of the style, but wanted to ask before merge

Reply:
I disagree with this `max` value, I think that _strictly_ limiting to 2 lines is a bit too much. I usually allow up to 5 lines in my ESLint config to prevent people from going crazy with more lines than unnecessary. But I've found situations where I want to add up to 4~5 empty lines to make it visually clear that pieces of code are separated in the same file.